msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md

```diff
@@ -0,0 +1,236 @@
+
+1. Meeting API URL parameters may require `meetingId`, `userId`, and the `[tenantId`](/onedrive/find-your-office-365-tenant-id) These are available as part of the Teams Client SDK. Additionally, reliable information for user ID and tenant ID can be retrieved when the Tab uses SSO authentication.
+
+1. Some meeting APIs, such as `GetParticipant` will require a [bot registration and bot app ID](../bots/how-to/create-a-bot-for-teams.md#with-an-azure-subscription) to generate auth tokens.
+
+1. Adherence to the [Teams tab design guidelines](../tabs/design/tabs.md) is a best practice for tabs.
+
+## Meeting apps API reference
+
+|API|Description|Request|Source|
+|---|---|----|---|
+|**GetParticipant**|This API allows a bot to fetch a participant information by meeting id and participant id.|**GET** _**/v1/meetings/{meetingId}/participants/{participantId}?tenantId={tenantId}**_ |Microsoft Bot Framework SDK|
+|**NotificationSignal** |Meeting signals will be delivered using the following existing conversation notification API (for user-bot chat). This API allows developers to signal based on end-user action to show-case an in-meeting dialog bubble.|**POST** _**/v3/conversations/{conversationId}/activities**_|Microsoft Bot Framework SDK|
+|**GetUserContext**| Get contextual information to display relevant content in a Teams tab. |_**microsoftTeams.getContext( ( ) => {  /*...*/ } )**_|Microsoft Teams client SDK|
+
+### GetParticipant API
+
+#### Request
+
+```http
+GET /v1/meetings/{meetingId}/participants/{participantId}?tenantId={tenantId}
+```
+
+#### Query parameters
+
+**meetingId**. The meeting identifier is required.  
+**participantId**. The participant identifier is required.  
+**tenantId**. Tenant id of the participant. Required for tenant user.
+
+#### Response Payload
+<!-- markdownlint-disable MD036 -->
+
+**Example 1**
+
+```json
+{
+    "meetingRole":"<meeting role>",
+    "conversation":
+        {
+            "id": "<conversation id>"
+        }
+}
+```
+
+**meetingRole** can be *organizer*, *presenter*, or *attendee*.
+
+**Example 2**
+
+```json
+{
+    "meetingRole": "Attendee",
+    "conversation": {
+        "isGroup": true,
+        "id": "sample" }
+}
+```
+
+#### Response Codes
+
+**200**: participant information successfully retrieved  
+**401**: invalid token  
+**403**: the app is not allowed to get participant information. There can be many reasons: app disabled by tenant admin, blocked during live site mitigation, etc.    
+**404**: the meeting doesn't exist or participant can’t be found    
+
+<!-- markdownlint-disable MD024 -->
+### NotificationSignal API
+
+#### Request
+
+```http
+POST /v3/conversations/{conversationId}/activities
+```
+
+#### Query parameters
+
+**conversationId**: The conversation identifier. Required
+
+#### Request Payload
+
+**Example 1**
+
+```json
+{
```

Comment:
This JSON is incomplete, right? The app must post a full activity, and that activity is sent to the conversation in addition to the bubble showing up.

(The docs have given the impression that you can trigger a bubble alone, without a message to the meeting chat, which isn't the case.)